.github/workflows/pr_checklist.yml

```diff
@@ -0,0 +1,33 @@
+name: PR Checklist
+
+on:
+  pull_request:
+    types: [opened]
+
+jobs:
+  add-checklist:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Comment PR with checklist
+        uses: peter-evans/create-or-update-comment@v3
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          issue-number: ${{ github.event.pull_request.number }}
+          body: |
+            Thank you for submitting your PR. The PR states are In progress (or Draft) -> Tech review -> Doc review -> Editorial review -> Merged.
+            
+            Before you mark the PR ready for doc review, make sure the content is technically accurate. If you need help finding a tech reviewer, tag a [maintainer](https://github.com/opensearch-project/documentation-website/blob/main/MAINTAINERS.md).
```

Comment:
Should we indicate how/in what way they would "mark" a PR as "ready"? I think we can be a little more precise here.

Reply:
How about "before the PR is ready for doc review"?